src/dhwcalc.cpp

```diff
@@ -4663,8 +4688,8 @@ RC DHWHEATER::wh_DoSubhrEnd(		// end-of-subhour
 	{
 		wh_HPWH.hw_DoSubhrEnd(mult, wh_pZn, wh_pAshpSrcZn);
 
-		wh_qLoss = wh_HPWH.hw_qLoss;
-		wh_qEnv = wh_HPWH.hw_qEnv;
+		wh_qLoss = BtuperkWh * wh_HPWH.hw_qLoss;	// tank loss, + = to surround
+		wh_qEnv = BtuperkWh * wh_HPWH.hw_qEnv;
```

Comment:
Doesn't seem relevant to this PR.

Reply:
It is not.  Work on DHW energy balance became entangled.